pkg/apis/alerting_notifications/v0alpha1/types_ext.go

```diff
@@ -0,0 +1,46 @@
+package v0alpha1
+
+const ProvenanceStatusAnnotationKey = "grafana.com/provenance"
+const ProvenanceStatusNone = "none"
```

Comment:
Will using the value "none" instead of empty string to represent `ProvenanceStatusNone` for k8s representations cause problems when switching between storages?

Reply:
the main idea of having a value for "none" status is to be able to filter by it. The meaning of filter by empty string is not obvious.
Another reason is that in K8s we represent it as an item in a dictionary, and heaving meaningful, self-explanatory value for a key seems to be a little bit better, in my opinion.